frontend/benefit/handler/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@frontend/bf-handler",
```

Comment:
It's good idea to distinct the name with `bf-handler` because there will be `ks-handler` for kesäseteli at some point